File: codewars/7kyu/summing-a-numbers-digits/dduck-summing-a-numbers-digits.py
# https://www.codewars.com/kata/52f3149496de55aded000410/train/javascript

from unittest import TestCase

sum_digits = lambda number: sum(list(map(int, str(abs(number)))))

TestCase().assertEqual(sum_digits(10), 1)
TestCase().assertEqual(sum_digits(99), 18)
TestCase().assertEqual(sum_digits(-32), 5)

File: codewars/7kyu/summing-a-numbers-digits/dduck-summing-a-numbers-digits.js
// https://www.codewars.com/kata/52f3149496de55aded000410/train/javascript

const sumDigits = (number) => `${Math.abs(number)}`.split('').reduce((a, c) => a + parseInt(c, 10), 0);

console.log(sumDigits(10), 1);
console.log(sumDigits(99), 18);
console.log(sumDigits(-32), 5);
